packages/integrations/src/rewriteframes.ts

```diff
@@ -97,7 +97,7 @@ export class RewriteFrames implements Integration {
           // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
           values: event.exception!.values!.map(value => ({
             ...value,
-            stacktrace: this._processStacktrace(value.stacktrace),
+            stacktrace: value.stacktrace ? this._processStacktrace(value.stacktrace) : undefined, // Avoid creating an empty stacktrace if undefined.
```

Comment:
This should make sure that it's only set if `value.stacktrace` is defined.

```suggestion
           ...(value.stacktrace && { stacktrace: this._processStacktrace(value.stacktrace) }),
```

We'll have to adjust the tests as well.